workshop/Lab0/README.md

```diff
@@ -62,7 +62,7 @@ To view a local version of the Kubernetes dashboard and to deploy apps into your
 1. Move the executable file to the `/usr/local/bin` directory using the command `mv /<path_to_file>/kubectl /usr/local/bin/kubectl` .
 
 2. Make sure that `/usr/local/bin` is listed in your PATH system variable.
-```
+```shell
 $echo $PATH
```

Comment:
This looks like an example of running a command in a terminal and showing the example output. Seems appropriate for there to be a `$` but needs to be a space between it and **echo**